todo/src/components/TodoList.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { connect } from "react-redux"; // Importing the connect, which is at the end of my code
+import { completed, addNew } from "../actions/action"; // importing function from actions.js to use
+import "../index.css";
+
+class TodoList extends Component {
+  // building the todolist component
+  state = {
+    // setting the state for my new todo to an empty string so i can manipulate
```

Comment:
loving all the comments so far, hope they're helping you out. Just make sure you're also reading them after a while. It's a great way to label stuff for yourself.